### PR TITLE
renewed PR automation for performance and clarity issues

### LINKED ISSUE #894: Rendertron container failing to start due to seccomp file not found
Hello,

Startup of intelowl_rendertron containers fails as the seccomp file cannot be located.
```
root@intelowl:/opt/IntelOwl# python3 start.py --all_analyzers prod up
[+] Running 12/13
 ⠿ Network intel_owl_default                 Created                                                                                                        0.0s
 ⠿ Container intelowl_rabbitmq               Created                                                                                                        0.0s
 ⠹ Container intelowl_rendertron             Creating                                                                                                       0.2s
 ⠿ Container intelowl_postgres               Created                                                                                                        0.0s
 ⠿ Container intelowl_uwsgi                  Created                                                                                                        0.0s
 ⠿ Container intelowl_apk_analyzers          Created                                                                                                        0.1s
 ⠿ Container intelowl_nginx                  Created                                                                                                        0.1s
 ⠿ Container intelowl_boxjs                  Created                                                                                                        0.1s
 ⠿ Container intelowl_qiling                 Created                                                                                                        0.1s
 ⠿ Container intelowl_celery_beat            Created                                                                                                        0.1s
 ⠿ Container intelowl_static_analyzers       Created                                                                                                        0.1s
 ⠿ Container intelowl_thug                   Created                                                                                                        0.1s
 ⠿ Container intelowl_celery_worker_default  Created                                                                                                        0.1s
opening seccomp profile ("integrations/rendertron/seccomp/chrome.json") failed: open /opt/IntelOwl/docker/"integrations/rendertron/seccomp/chrome.json": no such file or directory
```

Changing compose.yml in integrations/rendertron to read 
```
- seccomp:../integrations/rendertron/seccomp/chrome.json
```

instead of 
```
- seccomp:"/integrations/rendertron/seccomp/chrome.json"
```
fixed the issue for me.

### PR DESCRIPTION
**Pull request by mlodic**
fixes #894

### CODE REVIEW

File: tests/test_generic.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ def test_tor_updater(self):
         db_file_path = tor.Tor.updater()
         self.assertTrue(os.path.exists(db_file_path))
 
+
+class CronTestsYara(TestCase):
```
Comment (eshaan7) [inline tests/test_generic.py:45]: I understand why you made this change but I'd say for now we revert this and implement a proper solution later that also covers the tor updater, maxmind updater, etc. methods.

File: integrations/apk_analyzers/compose-tests.yml [hunk 2]
```diff
@@ -9,13 +9,4 @@ services:
...
-    container_name: intelowl_apk_analyzers
-    restart: unless-stopped
-    expose:
-      - "4004"
-    env_file:
-      - env_file_integrations
-    volumes:
-      - generic_logs:/var/log/intel_owl
-    depends_on:
-      - uwsgi
```
Comment (eshaan7) [inline integrations/apk_analyzers/compose-tests.yml:21]: why delete all these attributes from the configuration ?

Comment (mlodic) [reply integrations/apk_analyzers/compose-tests.yml:21]: I removed duplicated code between compose files and adjusted start.py script to properly leverage override files.

File: tests/test_generic.py (hunk 1 above)
Comment (mlodic) [reply tests/test_generic.py:45]: My main focus in this PR is to have a more proper coverage and management of the Docker Analyzers. I think that, with this new version, it is at least the third time that we make a release and the docker analyzers do not work properly.

So, I took that opportunity to rework the PR automation and also trying to separate cases between master and develop to speed up the CI.

This is a fast fix to have the tests work correctly with the REPO_DOWNLOADER_ENABLED=False enabled that would save us time and will avoid the abuse of that external service. This is extremely important to fix

Comment (mlodic) [reply tests/test_generic.py]: anyway I have removed that specific part (REPO_DOWNLOADER_ENABLED) because it would have failed too many analyzers.

**Pull request merged**